docs/guides/nextjs.md

```diff
@@ -224,11 +320,15 @@ export default function Home() {
 #### App Router
 
 ```tsx
-// components/pages/home-page.tsx
-import { useCounterStore } from 'components/counter-store-provider.ts'
+// src/components/pages/home-page.tsx
+'use client'
+
+import { useCounterStore } from '@/providers/counter-store-provider'
 
 export const HomePage = () => {
-  const { count, incrementCount, decrementCount } = useCounterStore()
+  const { count, incrementCount, decrementCount } = useCounterStore(
+    (state) => state,
```

Comment:
I don't mind much, but we can make it so that it accepts undefined selector (and use `identity` as default).

Reply:
@dai-shi sure